src/runtime-rs/crates/resource/src/rootfs/virtual_volume.rs

```diff
@@ -24,7 +24,7 @@ use kata_types::{
 /// Image guest-pull related consts
 const KUBERNETES_CRI_IMAGE_NAME: &str = "io.kubernetes.cri.image-name";
 const KUBERNETES_CRIO_IMAGE_NAME: &str = "io.kubernetes.cri-o.ImageName";
-const KATA_VIRTUAL_VOLUME_TYPE_OVERLAY_FS: &str = "overlayfs";
+const KATA_VIRTUAL_VOLUME_TYPE_OVERLAY_FS: &str = "overlay";
```

Comment:
Did the corresponding definition in agent  need changed?

Reply:
As analysis of the related codes and tests, I think it's a useless value for kata-agent.  I will raise another PR to try to cleanup such values.

Reply:
Virtual volumes are translated to full `agent.Storage` messages, which include the `fstype`. Theoretically the agent could use this field, but at the moment there's only one virtual storage handler, which just delegates the mounting to CDH and ignores most `Storage` fields.

https://github.com/kata-containers/kata-containers/blob/e3aa97399547d05d3d3ed9fc6bda539c9d5b6018/src/agent/src/storage/image_pull_handler.rs#L49